EXPO/native-base-theme/components/Form.js

```diff
@@ -0,0 +1,90 @@
+// @flow
```

Comment:
we can use file naming conventions for such fil as `Form.style.js` and for component `Form.js`

Reply:
Above code is in Native base theme third party Library.